### ISSUE TITLE
how to generate a image with same label?

### ISSUE THREAD

**Issue by Johnson-yue** [sangwoomo/FreezeD#2]
Hi,  you idea is very useful and interesting .

as I know the style GAN is  non-conditional generate model.So how to get an image with same label  
like in ReadMe.md file ?
an image have all eagle,  an image have all panda ....

How to do it ?  

Did you only use one class data to finetune styleGAN model ?? or use some other tricks

**Comment by sangwoomo** [comment]
Hi Johnson, 

I fine-tuned one model for each class.

One may train all classes at once then find the corresponding z for each class (see [1]), but I simply did it since my goal was to train a task-specific GAN rather than a general model for all possible images.

[1] Engel et al. Latent Constraints: Learning to Generate Conditionally from Unconditional Generative Models. ICLR 2018.

**Comment by Johnson-yue** [comment]
Thank you, I understand what you do.

**Comment by qinenergy** [comment]
Hi,
Does this mean that we only use the images from that specific class for finetuning? Something like this?
```
python prepare_data.py --out dataset/cat_lmdb --n_worker 8 dataset/animal/cathead
```


Thanks!

**Comment by sangwoomo** [comment]
@qinenergy You are exactly right :)

**Comment by qinenergy** [comment]
Thanks!
